src/inrange.jl

```diff
@@ -104,3 +104,60 @@ function inrangecount(tree::NNTree{V}, point::AbstractMatrix{T}, radius::Number)
     end
     return inrangecount(tree, new_data, radius)
 end
+
+"""
+    inrange_runtime!(tree::NNTree{V}, points::AbstractVector{T}, radius::Number, runtime_function::Function)
+
+Compute a runtime function for all in range queries.
+Instead of returning the indicies, the `runtime_function` is called for each point in points
+and is given the points, the index of the point, and the index of the neighbor.
+The `runtime_function` should return nothing.
+The `runtime_function` should be of the form:
+runtime_function(point_index::Int, neighbor_index::Int, point::AbstractVector{T})
+where `point_index` is the index of the point in `points`, `neighbor_index` is the index of the neighbor in the tree,
+and `point` is the point in points.
+
+The `runtime_function` should not modify the tree or the points.
+
+Ananymous functions can be used as well.
```

Comment:
No need to add this, in any case a function can be used an anonymous function can be used as well so it is just general julia knowledge which we don't have to document here.